src/Framework/PropertyInitialValueSetEventArgs.cs

```diff
@@ -59,6 +91,15 @@ public PropertyInitialValueSetEventArgs(
         /// </summary>
         public string PropertySource { get; set; }
 
+        public override string Message
+        {
+            get
+            {
+                var formattedSource = File != null ? $"{File} ({LineNumber},{ColumnNumber})" : PropertySource;
```

Comment:
we probably want to cache this in case Message gets called multiple times, need to see what other EventArgs classes do when they override Message

Reply:
We could get the string from resources here, since we know what string we want.